CommonTools/RecoAlgos/python/HBHENoiseFilterResultProducer_cfi.py

```diff
@@ -6,13 +6,10 @@
     minHPDHits = cms.int32(17),
     minHPDNoOtherHits = cms.int32(10),
     minZeros = cms.int32(9999),
-    IgnoreTS4TS5ifJetInLowBVRegion = cms.bool(True),
-    defaultDecision = cms.string("HBHENoiseFilterResultRun1"),
+    IgnoreTS4TS5ifJetInLowBVRegion = cms.bool(False),
+    defaultDecision = cms.string("HBHENoiseFilterResultRun2Loose"),
     minNumIsolatedNoiseChannels = cms.int32(10),
     minIsolatedNoiseSumE = cms.double(50.0),
-    minIsolatedNoiseSumEt = cms.double(25.0)
+    minIsolatedNoiseSumEt = cms.double(25.0),
+    useBunchSpacingProducer = cms.bool(True)
 )
-
-from Configuration.StandardSequences.Eras import eras
-eras.run2_common.toModify(HBHENoiseFilterResultProducer, IgnoreTS4TS5ifJetInLowBVRegion=False)
```

Comment:
I think this (IgnoreTS4TS5ifJetInLowBVRegion) is still useful and should stay.
My request was towards removing configuration-level settings to switch between 50ns or 25 ns in the same major period of data taking as we had in 2015

run1 vs run2 configuration changes should be OK for some time (trying to keep this down to detector level changes).